main/openssh/APKBUILD

```diff
@@ -39,32 +39,22 @@ source="http://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/$pkgname-$_myver.tar
 
 # HPN patches are from: http://www.psc.edu/index.php/hpn-ssh
 
-builddir="$srcdir"/$pkgname-$_myver
-prepare() {
-	cd "$builddir"
-	default_prepare
-	sed -i -e '/_PATH_XAUTH/s:/usr/X11R6/bin/xauth:/usr/bin/xauth:' \
-		pathnames.h || return 1
-}
-
 build() {
 	cd "$builddir"
-	export LD="$CC"
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
 		--sysconfdir=/etc/ssh \
-		--datadir=/usr/share/openssh \
 		--libexecdir=/usr/lib/ssh \
 		--mandir=/usr/share/man \
 		--with-pid-dir=/run \
 		--with-mantype=man \
 		--with-ldflags="${LDFLAGS}" \
-		--disable-strip \
 		--disable-lastlog \
+		--disable-strip \
 		--disable-wtmp \
-		--with-privsep-path=/var/empty \
```

Comment:
why rely on default privsep path? You later use a hardcoded path in `start_pre`. For that reason I think it might make sense to explicitly set it hard here too.